ratatui-widgets/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+#![warn(missing_docs)]
+//! `ratatui-widgets` is a collection of types that implement [`Widget`] or [`StatefulWidget`] or
+//! both.
+//!
+//! Widgets are created for each frame as they are consumed after rendered. They are not meant to be
+//! stored but used as *commands* to draw common figures in the UI.
+//!
+//! The available widgets are:
+//! - [`Block`]: a basic widget that draws a block with optional borders, titles and styles.
+//! - [`BarChart`]: displays multiple datasets as bars with optional grouping.
+//! - [`calendar::Monthly`]: displays a single month.
+//! - [`Canvas`]: draws arbitrary shapes using drawing characters.
+//! - [`Chart`]: displays multiple datasets as a lines or scatter graph.
+//! - [`Clear`]: clears the area it occupies. Useful to render over previously drawn widgets.
+//! - [`Gauge`]: displays progress percentage using block characters.
+//! - [`LineGauge`]: display progress as a line.
+//! - [`List`]: displays a list of items and allows selection.
+//! - [`Paragraph`]: displays a paragraph of optionally styled and wrapped text.
+//! - [`Scrollbar`]: displays a scrollbar.
+//! - [`Sparkline`]: display a single data set as a sparkline.
+//! - [`Table`]: displays multiple rows and columns in a grid and allows selection.
+//! - [`Tabs`]: displays a tab bar and allows selection.
+//!
+//! [`Canvas`]: crate::widgets::canvas::Canvas
+pub mod barchart;
+pub mod block;
+pub mod borders;
+pub mod canvas;
+pub mod chart;
+pub mod clear;
+pub mod gauge;
+pub mod list;
+pub mod logo;
+pub mod paragraph;
+pub mod scrollbar;
+pub mod sparkline;
+pub mod table;
+pub mod tabs;
+
+/// Internal module for reflowing text.
+mod reflow;
+
+#[cfg(feature = "calendar")]
+pub mod calendar;
+
+// use ratatui_core::widgets::Widget;
```

Comment:
Not needed?
```suggestion
```

Reply:
https://github.com/ratatui/ratatui/pull/1474/commits/62a9eeb24202c68e0a0ef2094ddb650d6ca93140